### PR TITLE
Create ArchitecturesController#index spec

### PR DESCRIPTION
**Pull request by mdeniz**
We missed to cover the index action in the specs

### CODE REVIEW

**Comment by ChrisBr** [comment]
Hey @mdeniz, why not splitting up your commits? The changes to 'POST #bulk_update_availability' are completely unrelated to your commit and also not covered in the commit message ...

**Comment by mdeniz** [comment]
@ChrisBr Splitted... please re-review

File: src/api/spec/controllers/webui/architectures_controller_spec.rb [hunk 1]
```diff
@@ -6,10 +6,17 @@
...
+  describe 'GET #index' do
+    before do
+      get :index
+    end
+
+    it { expect(assigns(:architectures)).to match_array(Architecture.all) }
+  end
 
         post :bulk_update_availability, params: { archs: { i586: '0', s390x: '1' } }
         expect(response).to redirect_to(architectures_path)
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/architectures_controller_spec.rb]: The indentation doesn't seem to fit anymore

Comment (mdeniz) [inline src/api/spec/controllers/webui/architectures_controller_spec.rb]: You already commented in the outdated commit... can you re-review it?

Comment (bgeuken) [inline src/api/spec/controllers/webui/architectures_controller_spec.rb]: Looks like it get's fixed in ec6f8318691b330fb1ff7d64b9eefa84a91f68ed . But would be nice to do that directly in 59ed5471b174da2e886b838475123b7212163780 , eg. via git rebase -i 59ed5471b174da2e886b838475123b7212163780^

Comment (mdeniz) [inline src/api/spec/controllers/webui/architectures_controller_spec.rb]: Done

**Pull request merged**